src/ToolBox/superpmi/superpmi-shim-collector/superpmi-shim-collector.cpp

```diff
@@ -84,10 +84,10 @@ void SetLogFilePath()
 
 extern "C"
 #ifdef FEATURE_PAL
-DLLEXPORT // For Win32 PAL LoadLibrary emulation
+    DLLEXPORT // For Win32 PAL LoadLibrary emulation
```

Comment:
VS editor insists on this reformatting because of https://github.com/dotnet/coreclr/blob/master/src/ToolBox/superpmi/.clang-format